studio/components/grid/components/editor/SelectEditor.tsx

```diff
@@ -19,11 +19,11 @@ export function SelectEditor<TRow, TSummaryRow = unknown>({
 
   const value = row[column.key as keyof TRow] as unknown as string
 
-  function onChange(value: string) {
+  function onChange(value: any) {
```

Comment:
Small nit: should this be called `event` instead of `value`? It seems a little odd to be saying `value.target.value` below.

Reply:
@alaister - Just renamed the `value` parameter as that does look odd.

Also fixed an issue where the `NULL` option was not saving when selected.

Reply:
Nice one! Just one small typo left and then we'll get this merged – thank you!